script/config/80084/42420/targetContractSetConfig.json

```diff
@@ -42,6 +35,29 @@
       "decimals": 18,
       "description": "sfrxETH/frxETH",
       "feedId": 42
+    }, 
+    {
+      "base": "0xdAC17F958D2ee523a2206206994597C13D831ec7",
+      "quote": "0x0000000000000000000000000000000000000348",
+      "feedId": 17,
+      "decimals": 18,
+      "description": "USDT/USD"
+    }, 
+    {
+      "base": "0xA0b86991c6218b36c1d19D4a2e9Eb0cE3606eB48",
+      "quote": "0x0000000000000000000000000000000000000348",
+      "feedId": 54,
+      "decimals": 6,
```

Comment:
Why 6 decimals?
We currently support only 18 decimals AFAIK

Reply:
my bad I assumed this was the base asset decimals